homeassistant/components/fibaro/__init__.py

```diff
@@ -321,14 +365,14 @@ def _read_scenes(self):
             device.ha_id = (
                 f"scene_{slugify(room_name)}_{slugify(device.name)}_{device.id}"
             )
-            device.unique_id_str = f"{self.hub_serial}.scene.{device.id}"
+            device.unique_id_str = f"{slugify(self.hub_serial)}.scene.{device.id}"
```

Comment:
why slugify? this becomes a breaking change now as unique IDs no longer match?

Reply:
I removed slugify from the member because I need it for the hub device info. So slugify is just moved from another place to here so no change of the unique id.